base/util_testing.go

```diff
@@ -320,3 +323,24 @@ func DirExists(filename string) bool {
 	}
 	return info.IsDir()
 }
+
+// getTestKeyNamespace returns a unique doc key namespace that can be prepended in tests.
+func getTestKeyNamespace(t *testing.T) string {
```

Comment:
This function has been removed from master.  It looks like you merged master into the oidc-coverage branch, instead of rebasing oidc-coverage on master.